lib/cli/src/frameworks/lit/ts/Button.ts

```diff
@@ -0,0 +1,91 @@
+import { LitElement, html, css } from 'lit';
+import { customElement } from 'lit/decorators.js';
+import { styleMap } from 'lit/directives/style-map.js';
+
+@customElement('sb-button')
+export class StorybookButton extends LitElement {
+  static get styles() {
+    return css`
+      .storybook-button {
+        font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+        font-weight: 700;
+        border: 0;
+        border-radius: 3em;
+        cursor: pointer;
+        display: inline-block;
+        line-height: 1;
+      }
+      .storybook-button--primary {
+        color: white;
+        background-color: #1ea7fd;
+      }
+      .storybook-button--secondary {
+        color: #333;
+        background-color: transparent;
+        box-shadow: rgba(0, 0, 0, 0.15) 0 0 0 1px inset;
+      }
+      .storybook-button--small {
+        font-size: 12px;
+        padding: 10px 16px;
+      }
+      .storybook-button--medium {
+        font-size: 14px;
+        padding: 11px 20px;
+      }
+      .storybook-button--large {
+        font-size: 16px;
+        padding: 12px 24px;
+      }
+    `;
+  }
+
+  // TODO: currently decorators are not reflected https://github.com/Polymer/lit-html/issues/1476
+  static get properties() {
+    return {
+      label: { type: String, reflect: true },
+      primary: { type: Boolean },
+      size: { type: String },
+      backgroundColor: { type: String },
+    };
+  }
+
+  primary?: boolean;
+
+  backgroundColor?: string;
+
+  size?: 'small' | 'medium' | 'large';
```

Comment:
Can you set the default size to medium here.  It's a bit clearer as to what the defaults are instead of needing to read the render function.